### PR TITLE
Added Paper to the native section

### PR DESCRIPTION
**Pull request by BomBardyGamer**

### CODE REVIEW

**Comment by MiniDigger** [comment]
https://github.com/KyoriPowered/adventure-docs/pull/20/files#diff-75939b0202027c39b05f793631c49ff5606f07c85572f42e57c19ffd9e496fd7R7 should be change too I guess

**Comment by BomBardyGamer** [comment]
Ah yes, will change now, thanks Mini

File: source/platform/native.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Native Support
 ==============
 
-As of this writing, the under-development Velocity 1.1.0 and Sponge 8 support Adventure
+As of this writing, the under-development Velocity 1.1.0, Sponge 8 and Paper 1.16.5 support Adventure
```
Comment (kashike) [inline source/platform/native.rst:7]:
```suggestion
As of this writing, the Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ support Adventure
```

**Review by kashike — CHANGES_REQUESTED** [review]

File: source/platform/native.rst (hunk 1 above)
Comment (MiniDigger) [reply source/platform/native.rst:7]: maybe its my dum english, but that `the` in there sounds dum if you remove the under development

Comment (AlexProgrammerDE) [reply source/platform/native.rst:7]: Sounds weird for me too. Either remove the `the` or add platforms/server softwares in the end. So like `As of this writing, the Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ platforms` or `As of this writing, the Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ server softwares`.

**Comment by BomBardyGamer** [comment]
I think my wording makes the most sense.

File: source/platform/native.rst [hunk 2]
```diff
@@ -4,7 +4,7 @@
 Native Support
 ==============
 
-As of this writing, the under-development Velocity 1.1.0 and Sponge 8 support Adventure
+As of this writing, Velocity 1.1.0+, Sponge 8+ and Paper 1.16.5+ support Adventure
```
Comment (kashike) [inline source/platform/native.rst:7]:
```suggestion
As of this writing, Velocity 1.1.0+, Sponge 8+, and Paper 1.16.5+ support Adventure
```

**Review by kashike — CHANGES_REQUESTED** [review]

**Review by kashike — APPROVED** [review]

**Review by zml2008 — APPROVED** [review]

**Pull request merged**